README.md

```diff
@@ -110,5 +151,38 @@ Thanks goes to these wonderful people
 </table>
 
 This project follows the [all-contributors](https://github.com/saigon-technology/rn-base-project/graphs/contributors) specification. Contributions of any kind welcome!
+## Dependencies
+
+```text
+    - react-native-mmkv
+    - @react-navigation/bottom-tabs
+    - @react-navigation/native
+    - @react-navigation/native-stack
+    - react-native-code-push
+    - react-native-fast-image
+    - react-native-gesture-handler
+    - react-native-progress
+    - react-native-reanimated
+    - react-native-safe-area-context
+    - react-native-screens
+    - react-redux
+    - redux-saga
+    - i18n-js
+    - plop
+```
+
+## mmkv storage
+
+- [Config mmkv](/docs/mmkv.md)
+
+## Networking
+
+- [Networking](/docs/networking.md)
+
+## Contributing
+
+Pull requests are welcome. For major changes, please open an issue first to discuss what you would like to change.
+
+Please make sure to update tests as appropriate.
```

Comment:
I think we removed these sections above, they are replace with more complete contents